blocks/table/table.css

```diff
@@ -131,3 +145,63 @@ main .table table tbody tr:nth-child(even) {
     visibility: visible;
   }
 }
+
+main .table.weather table {
+  padding: 30px;
+  border: none;
+  font-size: 2vw;
+}
+
+main .table.weather tr th:first-child {
+  border-right: 5px solid var(--color-navy);
```

Comment:
is this intentional? 

<img width="304" alt="Screen Shot 2023-04-28 at 12 57 51 PM" src="https://user-images.githubusercontent.com/43383503/235230851-efbb1a15-c8e9-46dd-94a1-ad44c3975fc5.png">

Reply:
Yes, this was in the provided design.